hw1/02-palindrome.html

```diff
@@ -6,20 +6,39 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
+
+    <style>
+      body {
+        background-color: rgb(0, 108, 250);
+      }
+
+      .container {
```

Comment:
consider using bootstrap's built-in container

Reply:
I think all of this can actually be done in bootstrap. It'd be less code, quicker dev time, and really easy to implement. I'd recommend reading the docs on bootstrap linked on the homework page or googled.